docs/tutorials/reporting.md

```diff
@@ -33,188 +35,97 @@ By default, the timestamp format of the output files is ISO 8601. This can be ch
 Prowler supports natively the following output formats:
 
 - CSV
-- JSON
 - JSON-OCSF
 - JSON-ASFF
-- HTML
 
 Hereunder is the structure for each of the supported report formats by Prowler:
 
-### HTML
-![HTML Output](../img/output-html.png)
-
 ### CSV
 
-CSV format has a set of common columns for all the providers, and then provider specific columns.
+CSV format has a set of common columns for all the providers.
 The common columns are the following:
 
-- ASSESSMENT_START_TIME
-- FINDING_UNIQUE_ID
+- AUTH_METHOD
+- TIMESTAMP
+- ACCOUNT_UID
+- ACCOUNT_NAME
+- ACCOUNT_EMAIL
+- ACCOUNT_ORGANIZATION_UID
+- ACCOUNT_ORGANIZATION_NAME
+- ACCOUNT_TAGS
+- FINDING_UID
 - PROVIDER
 - CHECK_ID
 - CHECK_TITLE
 - CHECK_TYPE
 - STATUS
 - STATUS_EXTENDED
+- MUTED
 - SERVICE_NAME
 - SUBSERVICE_NAME
 - SEVERITY
 - RESOURCE_TYPE
+- RESOURCE_UID
+- RESOURCE_NAME
 - RESOURCE_DETAILS
 - RESOURCE_TAGS
+- PARTITION
+- REGION
 - DESCRIPTION
 - RISK
 - RELATED_URL
 - REMEDIATION_RECOMMENDATION_TEXT
 - REMEDIATION_RECOMMENDATION_URL
-- REMEDIATION_RECOMMENDATION_CODE_NATIVEIAC
-- REMEDIATION_RECOMMENDATION_CODE_TERRAFORM
-- REMEDIATION_RECOMMENDATION_CODE_CLI
-- REMEDIATION_RECOMMENDATION_CODE_OTHER
+- REMEDIATION_CODE_NATIVEIAC
+- REMEDIATION_CODE_TERRAFORM
+- REMEDIATION_CODE_CLI
+- REMEDIATION_CODE_OTHER
 - COMPLIANCE
 - CATEGORIES
 - DEPENDS_ON
 - RELATED_TO
 - NOTES
+- PROWLER_VERSION
 
-And then by the provider specific columns:
-
-#### AWS
-
-- PROFILE
-- ACCOUNT_ID
-- ACCOUNT_NAME
-- ACCOUNT_EMAIL
-- ACCOUNT_ARN
-- ACCOUNT_ORG
-- ACCOUNT_TAGS
-- REGION
-- RESOURCE_ID
-- RESOURCE_ARN
-
-#### AZURE
-
-- TENANT_DOMAIN
-- SUBSCRIPTION
-- RESOURCE_ID
-- RESOURCE_NAME
+???+ note
+    Since Prowler v4 the CSV column delimiter is the semicolon (`;`)
 
-#### GCP
+In the previous Prowler version, each provider had some columns different from the rest. These are the cases that have changed:
```

Comment:
Can you move this section to the deprecations below? The same for all the providers.

Reply:
Done